src/sqlfluff/dialects/dialect_clickhouse.py

```diff
@@ -101,6 +101,26 @@
 )
 
 
+class BracketedArguments(BaseSegment):
```

Comment:
It could extend the ansi one and just override `match_grammar` but not need to override `type`? Same with a few other files.

Reply:
True, that might be slightly clearer. 👍